src/style/app.module.css

```diff
@@ -3612,6 +3708,9 @@ button[data-testid='createPostBtn'] {
     }
   }
 
+  .buttonList {
+    background-color: var(--light-blue) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`